gradle/scripts/ember.gradle

```diff
@@ -0,0 +1,45 @@
+apply plugin: "com.moowork.node"
+
+node {
+  // Version of node to use.
+  version = '7.8.0'
+
+  // Version of yarn to use.
+  yarnVersion = '0.21.3'
+
+  // Base URL for fetching node distributions (change if you have a mirror).
+  distBaseUrl = 'https://nodejs.org/dist'
+
+  // If true, it will download node using above parameters.
+  // If false, it will try to use globally installed node.
+  download = true
+
+  // Set the work directory for unpacking node
+  workDir = file("${project.buildDir}/nodejs")
+
+  // Set the work directory for NPM
+  yarnWorkDir = file("${project.buildDir}/ywarn")
+
+  // Set the work directory where node_modules should be located
+  nodeModulesDir = file("${project.projectDir}")
+}
+
+task bowerInstall(type: NodeTask, dependsOn: yarn) {
+  script = file('node_modules/bower/lib/bin/bower.js')
+  args = ['install']
+}
+
+task emberServe(type: NodeTask, dependsOn: bowerInstall) {
+  script = file('node_modules/ember-cli/bin/ember')
+  args = ['serve']
+}
+
+task emberBuild(type: NodeTask, dependsOn: bowerInstall) {
```

Comment:
Can we adds the production args to ember build?, the cli command is
 `ember build --environment production`
This will ensure the build output is in an optimal state and doesn't contain unnecessary development artifacts